test/run_portability_test.py

```diff
@@ -0,0 +1,199 @@
+"""Runs a single portability test from a test directory."""
+
+import argparse
+import datetime
+import glob
+import json
+import os
+import sys
+import time
+
+import requests
+
+# This is what we need to find in a test directory. I guess we'll glob the PR
+# test WDL itself since it's named after the pipeline.
+TEST_DIR_LAYOUT = {
+    "inputs": "test_inputs.json",
+    "dependencies": "dependencies.json",
+    "test": "*PR.wdl"
+}
+
+# These are the environment variables we're expecting to be able to submit a
```

Comment:
I'm of two minds here: you normally define these things at the tops of files, but you don't use them until after you're done testing the inputs, and I had to scroll back up to remind myself about what they are. If it were me I'd be tempted to move them down for readability.

Reply:
Do you have a preferred place to put it? References to them are a little bit scattered throughout the code.

Reply:
I was thinking line 104.

Reply:
Okay thanks, I'll move it down.